src/color.rs

```diff
@@ -2368,984 +2310,9 @@ mod tests {
-        "seafoam green", SEAFOAM_GREEN, (122.0, 249.0, 171.0, 1.0);
-        "blue green", BLUE_GREEN, (19.0, 126.0, 109.0, 1.0);
-        "khaki", KHAKI, (170.0, 166.0, 98.0, 1.0);
-        "burgundy", BURGUNDY, (97.0, 0.0, 35.0, 1.0);
-        "dark teal", DARK_TEAL, (1.0, 77.0, 78.0, 1.0);
-        "brick red", BRICK_RED, (143.0, 20.0, 2.0, 1.0);
-        "royal purple", ROYAL_PURPLE, (75.0, 0.0, 110.0, 1.0);
-        "plum", PLUM, (88.0, 15.0, 65.0, 1.0);
-        "mint green", MINT_GREEN, (143.0, 255.0, 159.0, 1.0);
-        "gold", GOLD, (219.0, 180.0, 12.0, 1.0);
-        "baby blue", BABY_BLUE, (162.0, 207.0, 254.0, 1.0);
-        "yellow green", YELLOW_GREEN, (192.0, 251.0, 45.0, 1.0);
-        "bright purple", BRIGHT_PURPLE, (190.0, 3.0, 253.0, 1.0);
-        "dark red", DARK_RED, (132.0, 0.0, 0.0, 1.0);
-        "pale blue", PALE_BLUE, (208.0, 254.0, 254.0, 1.0);
-        "grass green", GRASS_GREEN, (63.0, 155.0, 11.0, 1.0);
-        "aquamarine", AQUAMARINE, (4.0, 216.0, 178.0, 1.0);
-        "burnt orange", BURNT_ORANGE, (192.0, 78.0, 1.0, 1.0);
-        "neon green", NEON_GREEN, (12.0, 255.0, 12.0, 1.0);
-        "bright blue", BRIGHT_BLUE, (1.0, 101.0, 252.0, 1.0);
-        "rose", ROSE, (207.0, 98.0, 117.0, 1.0);
-        "light pink", LIGHT_PINK, (255.0, 209.0, 223.0, 1.0);
-        "mustard", MUSTARD, (206.0, 179.0, 1.0, 1.0);
-        "indigo", INDIGO, (56.0, 2.0, 130.0, 1.0);
-        "sea green", SEA_GREEN, (83.0, 252.0, 161.0, 1.0);
-        "periwinkle", PERIWINKLE, (142.0, 130.0, 254.0, 1.0);
-        "dark pink", DARK_PINK, (203.0, 65.0, 107.0, 1.0);
-        "olive green", OLIVE_GREEN, (103.0, 122.0, 4.0, 1.0);
-        "peach", PEACH, (255.0, 176.0, 124.0, 1.0);
-        "pale green", PALE_GREEN, (199.0, 253.0, 181.0, 1.0);
-        "light brown", LIGHT_BROWN, (173.0, 129.0, 80.0, 1.0);
-        "hot pink", HOT_PINK, (255.0, 2.0, 141.0, 1.0);
-        "lilac", LILAC, (206.0, 162.0, 253.0, 1.0);
-        "navy blue", NAVY_BLUE, (0.0, 17.0, 70.0, 1.0);
-        "royal blue", ROYAL_BLUE, (5.0, 4.0, 170.0, 1.0);
-        "salmon", SALMON, (255.0, 121.0, 108.0, 1.0);
-        "bright green", BRIGHT_GREEN, (1.0, 255.0, 7.0, 1.0);
-        "dark purple", DARK_PURPLE, (53.0, 6.0, 62.0, 1.0);
-        "mauve", MAUVE, (174.0, 113.0, 129.0, 1.0);
-        "forest green", FOREST_GREEN, (6.0, 71.0, 12.0, 1.0);
-        "aqua", AQUA, (19.0, 234.0, 201.0, 1.0);
-        "tan", TAN, (209.0, 178.0, 111.0, 1.0);
-        "dark blue", DARK_BLUE, (0.0, 3.0, 91.0, 1.0);
-        "turquoise", TURQUOISE, (6.0, 194.0, 172.0, 1.0);
-        "dark green", DARK_GREEN, (3.0, 53.0, 0.0, 1.0);
-        "violet", VIOLET, (154.0, 14.0, 234.0, 1.0);
-        "light purple", LIGHT_PURPLE, (191.0, 119.0, 246.0, 1.0);
-        "lime green", LIME_GREEN, (137.0, 254.0, 5.0, 1.0);
+    #[test]
+    fn check_all_color_lists() {
+        assert_eq!(Color::all_colors().len(), Color::all_color_names().len());
```

Comment:
Great idea to add this test! This is definitely a very important property. :)

Reply:
:slightly_smiling_face: